src/main/java/com/iexec/common/notification/TaskNotificationExtra.java

```diff
@@ -31,6 +31,7 @@
 public class TaskNotificationExtra {
 
     private WorkerpoolAuthorization workerpoolAuthorization;
+    private String smsUrl;
```

Comment:
There are strong similarities between `TaskNotificationExtra` and `ReplicateTaskSummary`.
There should be a possible improvement of the model here later.

Reply:
I agree!